sds/src/secondary_validation/github_token_checksum.rs

```diff
@@ -20,11 +20,14 @@ impl Validator for GithubTokenChecksum {
             return false;
         }
 
-        // extract the payload (everything except the last 6 chars)
         let computed_checksum = crc32fast::hash(last_part[..last_part.len() - 6].as_bytes());
-        let computed_checksum_b62 = base62::encode(computed_checksum);
-        // check that the crc is the last 6 chars
-        computed_checksum_b62 == last_part[last_part.len() - 6..]
+        let checksum = base62::decode(last_part[last_part.len() - 6..].as_bytes());
+
```

Comment:
@vinckama it would be great if you can find a non reg test...

Reply:
done in [a6a9439](https://github.com/DataDog/dd-sensitive-data-scanner/pull/156/commits/a6a9439e643fe249e6447696682aa10df6e49820)

Reply:
**suggestion:** 
Would be great here to separate the two parts of the `last_part` &str.

Create one variable called `checksum` and `encoded_checksum` that correspond to `last_part[..n-6]` and `last_part[n-6...]` so that it's clearer that the `last_part` is made of two chunks
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
‏